.gitignore

```diff
@@ -1,3 +1,4 @@
+package-lock.json
```

Comment:
This will cause this file to be ignored in all subdirectories, is that intentional?